src/scripts/main.js

```diff
@@ -1 +1,47 @@
 'use strict';
+
+const promise1 = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
+
+const promise2 = new Promise((resolve) => {
+  document.addEventListener('mousedown', () => {
+    resolve('Second promise was resolved');
+  });
+});
+
+const promise3 = new Promise((resolve) => {
+  document.addEventListener('click' && 'contextmenu', () => {
```

Comment:
how does it work?
you pass 'click' && 'contextmenu' returns  'contextmenu', so you set listener for `contextmenu` only
it doesn't suit the task description